SimG4Core/MagneticField/interface/Field.h

```diff
@@ -20,4 +24,26 @@ namespace sim {
     mutable double oldb[3];
   };
 };  // namespace sim
+
+void sim::Field::GetFieldValue(const G4double xyz[], G4double bfield[3]) const {
```

Comment:
@VinInn , sorry for trivial question: in new compilers virtual methods may be successfully inlined? Is there any advantage in CPU?

Reply:
it is indeed inlined (as I finalized and it is used as such thanks to the template)
I observed gain in performance

Reply:
It is because it is "final"? or recent compilers can understand inline virtual?

Reply:
because it is final and the caller has a pointer of type ```sim::Field*``` not of the parent.

Reply:
Good to know! We have number of run time places in Geant4 and may be few in CMSSW SIM, where this can be used.